### PR TITLE
Improve docs about video support

### PR DESCRIPTION
**Pull request by emilk**
We do support AV1 on native, and now the docs say so too.

* Closes https://github.com/rerun-io/rerun/issues/7662
 
### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7687?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7687?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide
* [x] I've read the docs preview at https://landing-e6yens1hp-rerun.vercel.app/docs/reference/video

- [PR Build Summary](https://build.rerun.io/pr/7687)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: docs/content/reference/video.md [hunk 1]
```diff
@@ -24,25 +24,30 @@ This gives the best compression ratio, reducing file sizes and bandwidth require
...
+* [#7594](https://github.com/rerun-io/rerun/issues/7594): HDR video is not supported
 * There is no video encoder in the Rerun SDK, so you need to create the video file yourself
-* Only a limited sets of codecs are supported (see below)
+* A limited sets of codecs are supported on web (see below)
 
-## Web viewer support
-As of writing, playback of `AssetVideo` is only supported on the web viewer.
-Native video playback is coming, and can be tracked [in this GitHub issue](https://github.com/rerun-io/rerun/issues/7298).
+## Streaming video
+Rerun does not yet support streaming video support. You can work around this limitation by logging many small `AssetVideo`s to the same Entity Path. See [#7484](https://github.com/rerun-io/rerun/issues/7484) for more.
```
Comment (Wumpf) [inline docs/content/reference/video.md:37]:
```suggestion
Rerun does not yet support streaming video support. For scenarios where you don't need live video, you can work around this limitation by logging many small `AssetVideo`s to the same Entity Path. See [#7484](https://github.com/rerun-io/rerun/issues/7484) for more.
```

I think we shouldn't pretend it's a replacement if you want live video streaming in. That's completely unfeasible with that approach and I'd rather don't have people try

File: docs/content/reference/video.md [hunk 2]
```diff
@@ -24,25 +24,30 @@ This gives the best compression ratio, reducing file sizes and bandwidth require
 snippet: archetypes/video_auto_frames
 
 ## `AssetVideo` limitations
```
Comment (Wumpf) [inline docs/content/reference/video.md:26]:
```suggestion
## Video playback limitations
```

File: docs/content/reference/video.md [hunk 3]
```diff
@@ -24,25 +24,30 @@ This gives the best compression ratio, reducing file sizes and bandwidth require
...
+
+## Web viewer support
+Video playback in the Rerun Web Viewer is done using the browser's own video decoder, so the supported codecs depend on your browser.
 
 Overall, we recommend using Chrome or another Chromium-based browser, as it seems to have the best video support as of writing.
 
 When choosing a codec, we recommend [AV1](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Video_codecs#av1),
 as it seems to have the best overall playback support while also having very high compression quality.
-Since AV1 is patent-free, it is also likely the first codec we will support in the native viewer.
+AV1 is also patent-free, and is the only codec we currently support in the native viewer (see [#7298](https://github.com/rerun-io/rerun/issues/7298)). H.264/avc is another popular choice, and native support for that is coming soon.
```
Comment (Wumpf) [inline docs/content/reference/video.md:50]: Suggestion: Let's add a `## Codec support` section with this block at the start and then subsection `### Native` & `Web viewer` under it
(is it `native` or `desktop`? I noticed some folks don't have `native` as a term ingrained we do coming from systems programming languages)

**Review by Wumpf — CHANGES_REQUESTED** [review]

File: crates/store/re_types/definitions/rerun/archetypes/asset_video.fbs [hunk 4]
```diff
@@ -2,10 +2,10 @@ namespace rerun.archetypes;
 
 /// A video binary.
 ///
-/// NOTE: Videos can only be viewed in the Rerun web viewer.
-/// Only MP4 containers with a limited number of codecs are currently supported, and not in all browsers.
+/// Only MP4 containers with AV1 is generally supported,
+/// though the web viewer sometimes supports some more video codex, depending on browser.
```
Comment (Wumpf) [inline crates/store/re_types/definitions/rerun/archetypes/asset_video.fbs:6]:
```suggestion
/// Only MP4 containers with AV1 are generally supported,
/// though the web viewer support more video codex, depending on browser.
```

I mean they support _a lot_ more and I don't think we'll add any more there ourselves :)

File: docs/content/reference/video.md (hunk 3 above)
Comment (Wumpf) [reply docs/content/reference/video.md:52]: I still think this section is native/web independent and should go directly under the top `## Codec support` 🤷

**Review by Wumpf — APPROVED** [review]

File: docs/content/reference/video.md (hunk 3 above)
Comment (emilk) [reply docs/content/reference/video.md:52]: :+1:

**Pull request merged**